### PR TITLE
Add pointer initializers

### PR DESCRIPTION
**Pull request by cmpute**

### CODE REVIEW

**Review by b1f6c1c4 — CHANGES_REQUESTED** [review]
An empty line after `#include` is required. Refer to the cpp files.

File: snippet/single/make-shared-explicit-initializer.hpp [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include <memory>
+template <typename T>
+T create()
+{
+    auto list = std::initializer_list<int>{57257};
+    return *std::make_shared<T>(list);
```
Comment (b1f6c1c4) [inline snippet/single/make-shared-explicit-initializer.hpp]: You may want to inline `list`

File: snippet/single/make-unique-explicit-initializer.hpp [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#include <memory>
+template <typename T>
+T create()
+{
+    auto list = std::initializer_list<int>{57257};
+    return *std::make_unique<T>(list);
```
Comment (b1f6c1c4) [inline snippet/single/make-unique-explicit-initializer.hpp]: You may want to inline `list`

**Review by b1f6c1c4 — CHANGES_REQUESTED** [review]
BTW, as for the file name, you should use `*-initializer` instead of `*-explicit-initializer`. See the naming of other snippets.

**Comment by cmpute** [comment]
I use the name "explicit initializer" since I declare initializer explicitly instead of using initializer syntax.

**Pull request merged**